tests/integration/test_charm.py

```diff
@@ -543,3 +543,92 @@ async def test_anonymize_user(
             timeout=5,
         )
     assert res.status_code == 403
+
+
+@pytest.mark.usefixtures("s3_backup_bucket")
+async def test_synapse_enable_s3_backup_integration_success(
+    model: Model,
+    synapse_app: Application,
+    s3_backup_configuration: dict,
+):
+    """
+    arrange: Synapse App deployed and s3-integrator deployed with bucket created.
+    act:  integrate s3-integrator with Synapse.
+    assert: Synapse gets into active status.
+    """
+    s3_integrator_app = await model.deploy(
+        "s3-integrator",
+        channel="latest/edge",
+        config={
```

Comment:
Couldn't the `s3_backup_configuration` just be passed here directly rather than re-mapping it?

Reply:
done!